File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/PlacementActAdm/ViewPagerAdapterPlacementAdm.java
package com.example.adminapp.PlacementActAdm;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

public class ViewPagerAdapterPlacementAdm extends FragmentPagerAdapter {


    public ViewPagerAdapterPlacementAdm(@NonNull FragmentManager fm) {
        super(fm);
    }

    @NonNull
    @Override
    public Fragment getItem(int position) {
        if(position==0){
            return new PlacementDefaultFAdm();
        }
        return new PlacementResourcesFAdm();
    }

    @Override
    public int getCount() {
        return 2;
    }


    @Override
    public CharSequence getPageTitle(int position) {
        if(position == 0){
            return "Placement Offers";
        }
        return "Resources";
    }
}

File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/PlacementActAdm/ViewPagerAdapterPlacementAdm.kt
package com.example.adminapp.PlacementActAdm

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter

class ViewPagerAdapterPlacementAdm(fm: FragmentManager) : FragmentPagerAdapter(fm) {
    override fun getItem(position: Int): Fragment {
        return if (position == 0) {
            PlacementDefaultFAdm()
        } else PlacementResourcesFAdm()
    }

    override fun getCount(): Int {
        return 2
    }

    override fun getPageTitle(position: Int): CharSequence? {
        return if (position == 0) {
            "Placement Offers"
        } else "Resources"
    }
}
